arangod/Aql/Query.cpp

```diff
@@ -1448,3 +1469,58 @@ aql::ExecutionState Query::cleanupTrxAndEngines(ErrorCode errorCode) {
     return ExecutionState::DONE;
   }
 }
+
+  void Query::debugKillQuery() {
+#ifndef ARANGODB_ENABLE_FAILURE_TESTS
+    TRI_ASSERT(false);
+    return;
+#else
+    if (_wasDebugKilled) {
+      return;
+    }
+    bool usingSystemCollection = false;
+    // Ignore queries on System collections, we do not want them to hit failure points
+    collections().visit([&usingSystemCollection](std::string const&, Collection& col) -> bool {
+      if (col.getCollection()->system()) {
+        usingSystemCollection = true;
+        return false;
+      }
+      return true;
+    });
+
+    if (usingSystemCollection) {
+      LOG_DEVEL << "Using system collection: " << std::boolalpha << usingSystemCollection;
```

Comment:
I am in favor of deleting this log line.
```suggestion
```

Reply:
aarggh!!!
Shame on me!